.github/workflows/build-libraries.yaml

```diff
@@ -60,18 +60,29 @@ jobs:
           printf 'Solution: %s\n' "$solution"
           printf 'Project: %s\n' "$project"
 
+          [ -f "$solution/global.json" ] || echo "{}" > "$solution/global.json"
+          settings=$(jq -srec '.[0] * .[1]' "global.json" "$solution/global.json")
+          sdk=$(printf '%s' "$settings" | jq -rce '.sdk.version')
+
           echo "SOLUTION=$solution" >> $GITHUB_ENV
           echo "PROJECT=$project" >> $GITHUB_ENV
+          echo "SDK_VERSION=$sdk" >> $GITHUB_ENV
 
       - name: Resolve version
         id: resolve-version
         shell: bash
         run: echo "result=$(yq '.version' ${{ env.SOLUTION }}/version.yaml)" >> $GITHUB_OUTPUT
 
+      - name: Pin dotnet version
+        shell: bash
+        working-directory: ${{ env.SOLUTION }}
+        run: echo '{"sdk":{"version":"${{ env.SDK_VERSION }}"}}' > global.json
```

Comment:
Should "rollFoward" be set to "disable" here?

Reply:
It is now.